src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -96,14 +96,24 @@ async def test_TC_RVCCLEANM_2_1(self):
             app_pid = self.matter_test_config.app_pid
             if app_pid == 0:
                 asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set")
-            self.app_pipe = self.app_pipe + str(app_pid)
```

Comment:
I think that this is needed as `write_to_app_pipe` is used later without specifying the `app_pipe_name`. See [here](https://github.com/project-chip/connectedhomeip/blob/f212d601738728d58e88ada3bc7e65b84cdfe7f0/src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py#L1028).

Reply:
Thank you @hicklin,
You are correct that was definitely needed in there and not sure how I had deleted it. 
However, I have returned it to the code now.